src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -928,7 +934,8 @@ public void registerListener(Object object) {
     public void unregisterListener(Object object) {
         try {
             this.eventBus.unregister(object);
-        } catch (IllegalArgumentException e) {
+        } catch (
+                IllegalArgumentException e) {
```

Comment:
Please undo

Reply:
@Toro520 If you have time, you can investigate the IntelliJ code style checks why this happens. It is a thing that annoys us, but we did not have found the patience to investigate.

Reply:
Huh,  I suppose I do not manipulate this 😢 . Perhaps I used the boss key ctrl + Alt + L, to organized the entire code format.

Reply:
@Toro520 Yes, you did. And in a "proper" IDE configuration, the code is well-formatted. We have an issue in our rules. Quick fix: Undo that change (you can do line-based stashing in git gui. Very helpful for these kind of issues). The real fix would be to fix our rules... But takes time at least 1 hour. But one learns much about IDE configuration and code style management in projects.